webpack/scenes/ContentViews/Details/Versions/Compare/CVVersionCompareHeader.js

```diff
@@ -0,0 +1,194 @@
+            <OutlinedQuestionCircleIcon />
+          </Button>
+        </Tooltip>
+      </GridItem>
+      <GridItem span={12}>
+        <Flex className="example-border">
+          <FlexItem>
+            <Flex direction={{ default: 'column' }}>
+              <TextContent>
+                <Text component={TextVariants.h4}>{__('Versions to compare')}</Text>
+              </TextContent>
+              <FlexItem>
+                <Flex className="example-border">
+                  <FlexItem>
+                    <Select
+                      style={{ marginRight: '10px' }}
+                      variant={SelectVariant.single}
+                      placeholderText="Select an option"
+                      aria-label="Select version one"
+                      onToggle={setIsOpenSelectVersionOne}
+                      onSelect={onSelectVersionOne}
+                      selections={versionOne}
+                      isOpen={isOpenSelectVersionOne}
+                      isDisabled={!filteredVersionsFirstSelect?.length}
+                    >
+                      {filteredVersions(versionTwo)?.map(({ version }) => (
+                        <SelectOption
+                          key={version}
+                          value={version}
+                        >
+                          {__('Version ')}{version}
+                        </SelectOption>
+                      ))}
+                    </Select>
+                  </FlexItem>
+                  <FlexItem>
+                    <TextContent style={{ margin: '10px' }}>
+                      <Text component={TextVariants.h4}>{__('to')}</Text>
+                    </TextContent>
+                  </FlexItem>
+                  <FlexItem>
+                    <Select
+                      variant={SelectVariant.single}
+                      placeholderText="Select an option"
+                      aria-label="Select version two"
+                      onToggle={setIsOpenSelectVersionTwo}
+                      onSelect={onSelectVersionTwo}
+                      selections={versionTwo}
+                      isOpen={isOpenSelectVersionTwo}
+                      isDisabled={!filteredVersionsSecondSelect?.length}
+                    >
+                      {filteredVersions(versionOne)?.map(({ version }) => (
+                        <SelectOption
+                          key={version}
+                          value={version}
+                        >
+                          {__('Version ')}{version}
+                        </SelectOption>
+                      ))}
+                    </Select>
+                  </FlexItem>
+                </Flex>
+              </FlexItem>
+            </Flex>
+          </FlexItem >
+          <FlexItem style={{ marginLeft: '60px' }}>
+            <Flex direction={{ default: 'column' }}>
+              <FlexItem>
+                <TextContent>
+                  <Text component={TextVariants.h4}>{__('View by')}</Text>
```

Comment:
Could you hide this in this PR with a simple {false && ...} maybe since this is not yet implemented and this page will go into nightly as soon as merged.